src/controllers/storeController.ts

```diff
@@ -1,5 +1,7 @@
-import Store from '../models/Store';
```

Comment:
We can just return the address in `/store/:id`, no point having another call separately for that.

Reply:
Ideally, all of the calls that get stores should return address inside also as it is a common property.

Reply:
👍